packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -168,3 +168,15 @@ ButtonTypes = () => (
   </React.Fragment>
 );
 ```
+
+```js title=Small
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+
+ButtonSmallVariants = () => (
+  <React.Fragment>
+    <Button variant="primary" isSmall>Primary</Button> <Button variant="secondary" isSmall>Secondary</Button>{' '}
+    <Button variant="tertiary" isSmall>Tertiary</Button> <Button variant="danger" isSmall>Danger</Button><br /><br />
```

Comment:
Small nit, but could each of these be on their own line? And are the breaks at the end necessary?

Reply:
Yes, sure!